### DISCUSSION TITLE
Help with File Explorer after uninstall

### DISCUSSION THREAD

**Discussion by cdaug16** [Open-Shell/Open-Shell-Menu#1984] [Q&A]
![Screenshot 2024-08-29 202520](https://github.com/user-attachments/assets/4f359f2e-0254-4c56-8d47-6f0be7b82ee9)

Running Windows 11, after uninstalling Open-Shell and Classic Explorer, now the first item in File Explorer is always highlighted when I open it (see Quick Access in screenshot). Any idea how to undo this? I think this shows up when using the keypad arrows to navigate in File Explorer, but it shows up whenever File Explorer first opens. I know it's a small thing, but I just want to get my computer back to how it was before installing. I already had another issue with uninstalling Classic Explorer [Issue #326](https://github.com/Open-Shell/Open-Shell-Menu/issues/326#issuecomment-813086363)

**Comment by gubsy420** [comment]
I am having this same issue. Did you ever resolve this?
